homework-submission/week2/src/ToDo/DoneCheckBox.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+let isChecked = false;
+
+const todos = [
+  {
+    "id": 1,
+    "description": "Get out of bed",
+    "deadline": "2017-09-11",
+    "done": true
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth",
+    "deadline": "2017-09-10",
+    "done": false
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast",
+    "deadline": "2017-09-09",
+    "done": false
+  }
+]
```

Comment:
you are using the state of this component and that is not required, you should pass the state from App component to here as props